config/jobs/image-pushing/k8s-staging-kubebuilder.yaml

```
@@ -0,0 +1,24 @@
+postsubmits:
+  kubernetes-sigs/kubebuilder:
+
+    - name: post-kube-rbac-proxy-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-api-machinery-kubebuilder, sig-k8s-infra-gcb
+      decorate: true
+      run_if_changed: 'TAG'
```

```suggestion
      run_if_changed: "^kube-rbac-proxy-releases/"
```

I see so it should be ^